en/guides/teachers/creating-a-course/index.md

```diff
@@ -67,19 +67,19 @@ Finally, you need to set a few things about the course itself, such as the name
 
   * `Only registered users`: only registered users can see the course in the courses overview. There is an icon to indicate to course members that other users cannot see the course on the course page itself. Only registered users can navigate to this page. Other users can only register for the course if they use a direct registration link.
 
-* `Access Link`: when creating a course, an access link is automatically generated as a shielding mechanism for hidden courses. Without this link, non-registered users cannot see the course page of a hidden course and therefore cannot register there. You can find this access link on the course edit page. Click the copy button next to the access link to copy the access link to the clipboard.
+* `Secret link`: when creating a course, a secret link is automatically generated as a shielding mechanism for hidden courses. Without this link, non-registered users cannot see the course page of a hidden course and therefore cannot register there. You can find this secret link on the course edit page. Click the copy button next to the access link to copy the access link to the clipboard.
```

Comment:
Shouldn't we switch the wording here? shielding -> access and something like With this link, user who normally can't see the course can get access. This can be used to give users access to hidden courses, or to give users from outside your institution access to your institution-only course. Maybe also add that these users will be able to register. The part about "click copy..." seems redundant to me.